### ISSUE TITLE
alt text 50 char limit

### ISSUE THREAD

**Issue by courdo1** [salsadigitalauorg/merlin-framework#49]
Images with an alt text larger than 50 characters fail to import correctly.

If an alt text description for an image exceeds 50 characters merlin should trim the value to the first 50 characters.
